issuing_transaction.go

```diff
@@ -40,10 +50,11 @@ type IssuingTransactionParams struct {
 // IssuingTransactionListParams is the set of parameters that can be used when listing issuing transactions.
 type IssuingTransactionListParams struct {
 	ListParams   `form:"*"`
-	Card         *string           `form:"card"`
-	Cardholder   *string           `form:"cardholder"`
-	Created      *int64            `form:"created"`
-	CreatedRange *RangeQueryParams `form:"created"`
+	Card         *string                     `form:"card"`
+	Cardholder   *string                     `form:"cardholder"`
+	Created      *int64                      `form:"created"`
+	CreatedRange *RangeQueryParams           `form:"created"`
+	Type         *IssuingTransactionListType `form:"type"`
```

Comment:
We don't use enums in params, this should be `*string` instead and we don't need the type. Also if we used the type we would use `IssuingTransactionType` instead

Reply:
Woot! Great to know.